lib/tapioca/generator.rb

```diff
@@ -63,7 +63,7 @@ def build_requires
 
       content = String.new
       content << rbi_header(
-        config.generate_command,
+        "tapioca require",
```

Comment:
It should be `bin/tapioca require` since we install the binstub right?

Reply:
Yeah, we should have all commands as `bin/tapioca xxx` now, since we know, for a properly initialized app, it should always work.